src/views/activity/components/transaction-card/transaction-card.view.tsx

```diff
@@ -4,19 +4,21 @@ import useTransactionCardStyles from "src/views/activity/components/transaction-
 import { ReactComponent as TransferL1Icon } from "src/assets/icons/l1-transfer.svg";
 import { ReactComponent as TransferL2Icon } from "src/assets/icons/l2-transfer.svg";
 import { ReactComponent as ReloadIcon } from "src/assets/icons/spinner.svg";
-import { ReactComponent as EthToken } from "src/assets/tokens/eth.svg";
-import { ReactComponent as DaiToken } from "src/assets/tokens/dai.svg";
 import Typography from "src/views/shared/typography/typography.view";
 import Card from "src/views/shared/card/card.view";
 import { getTimeFromNow, TimeFromNowParams } from "src/utils/time";
 import { convertTokenAmountToFiat } from "src/utils/amounts";
+import { useNavigate } from "react-router-dom";
+import routes from "src/routes";
+import TokenIcon from "src/views/shared/token-icon/token-icon";
 
-enum StatusText {
-  "on-hold" = "On Hold",
-  initiated = "Initiated",
-  processing = "Processing",
-  failed = "Error",
-}
+export const statusText = {
```

Comment:
Because enum must be in uppercase

Reply:
I don't get what you mean. Can you reformulate?

Reply:
He needs the keys in lowercase, and enum keys should be in uppercase and without hyphens. That's why we moved this to a constant

Reply:
Now I get it! I had to read [this thread](https://github.com/hermeznetwork/bridge-ui/pull/10#discussion_r829840403) to understand what you were talking about.

> As now this enum is shared I would write it in the domain module, next to the WalletName.

You're right here @elias-garcia, this enum belongs to the domain. And you're also right about the naming convention of the enums. I though disagree with the applied solution.

If we need to map a `Status` type with a `StatusText` enum, instead of converting the `enum` to an `object` to be able to use bracket notation, we create a function to do the job:

```typescript
type Status = "processing" | "initiated" | "on-hold" | "completed" | "failed";

export enum StatusText {
  ON_HOLD = "On Hold",
  INITIATED = "Initiated",
  PROCESSING = "Processing",
  FAILED = "Error",
  COMPLETED = "Completed",
}

export function getStatusText(status: Status): StatusText {
  switch (status) {
    case "completed":
      return StatusText.COMPLETED;
    case "failed":
      return StatusText.FAILED;
    case "initiated":
      return StatusText.INITIATED;
    case "on-hold":
      return StatusText.ON_HOLD;
    case "processing":
      return StatusText.PROCESSING;
  }
}
```

Reply:
Fixed.